docs/src/modify.md

```diff
@@ -31,4 +31,8 @@ using MarketData
 rename(cl, :Close′)
 rename(cl, [:Close′])
 rename(ohlc, [:Open′, :High′, :Low′, :Close′])
+rename(ohlc, :Open => :Open′)
+rename(ohlc, Dict(:Open => :Open′, :Close => :Close′))
+rename(Symbol ∘ uppercase ∘ string, ohlc)
+rename(uppercase, ohlc, asstring=true)
```

Comment:
I don't think this is a great API idea maybe we should use type dispatch instead and use

    rename(uppercase, ohlc, String)